app/assets/javascripts/authoring/awareness.js

```diff
@@ -594,8 +594,16 @@ var loadData = function(){
     googleDriveLink();
 };
 
+var retrieveSlackInfo = function(slack_info){
+    slackInfo = gon.slack_info;
+    flashTeamsJSON["slackToken"] = slackInfo.access_token;
+    flashTeamsJSON["slackChannel"] = slackInfo.incoming_webhook.channel;
+    flashTeamsJSON["slackURL"] = slackInfo.incoming_webhook.url;
+};
```

Comment:
Would be cleaner to put these details within `flashTeamsJSON["slackDetails"]` so we have flexibility to easily add more fields later without polluting the top-level name space of `flashTeamsJSON`.

For example,
```
flashTeamsJSON["slackDetails"] = {
    "slackToken": "sdfsdf",
    "slackChannel": "sdfds",
   ...
}
```